tests/braille-specs/hbo.yaml

```diff
@@ -13,16 +13,16 @@ table:
 flags: { testmode: forward }
 tests:
   # Hebrew
-  - ["שְׁמַ֖עa", "⠩⠄⠍⠉⠨⠞⠫⠁"]
+  - ["שְׁמַ֖עa", "⠩⠄⠍⠉⠨⠞⠫a"]
```

Comment:
This test was failing because it has the letter "a" in the expected output, instead of the braille dot 1. This change should fix it and get the tests to pass
```suggestion
  - ["שְׁמַ֖עa", "⠩⠄⠍⠉⠨⠞⠫⠁"]
```